systemd-stop-unit.bash

```diff
@@ -0,0 +1,61 @@
+#!/bin/sh
+
+# Inspired by: https://github.com/balena-io-experimental/balena-gateway-config
+
+function wait_for_dbus() {
+    echo "Waiting for DBus to be ready"
+	while true; do
```

Comment:
I'm okay with keeping it exactly in sync with the inspiration.

But otherwise you might as well directly evaluate the outcome of the command:

```bash
	until dbus-send --system \
			--print-reply \
			--dest=org.freedesktop.DBus \
			/org/freedesktop/DBus \
			org.freedesktop.DBus.ListNames
	do
		sleep 0.1
	done
```